src/javaExercises/primeNumbers/NotPrimeNumberExeption.java

```diff
@@ -0,0 +1,7 @@
+package javaExercises.primeNumbers;
+
+public class NotPrimeNumberExeption extends Exception{
```

Comment:
extend RuntimeException instead of Exception and check what is the difference between them